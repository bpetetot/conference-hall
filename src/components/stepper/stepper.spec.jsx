/* eslint-env jest */
import React from 'react'
import { shallow } from 'enzyme'
import Stepper from './index.js'

describe('components/stepper', () => {
  it('should render', () => {
    const steps = [
      { label: 'Event selection', icon: 'fa fa-bars' },
      { label: 'Talk details', icon: 'fa fa-microphone' },
      { label: 'Event submission', icon: 'fa fa-calendar-check-o' },
      { label: 'Done !', icon: 'fa fa-paper-plane' },
    ]
    const wrapper = shallow(<Stepper steps={steps} currentStep={0} />)
    expect(wrapper).toMatchSnapshot()
  })
})
