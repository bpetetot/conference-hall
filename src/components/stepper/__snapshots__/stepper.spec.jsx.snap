// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/stepper should render 1`] = `
<div
  className="stepper"
>
  <div
    className="steps-labels"
  >
    <StepLabel
      current={0}
      key="0"
      label="Event selection"
      step={0}
    />
    <StepLabel
      current={0}
      key="1"
      label="Talk details"
      step={1}
    />
    <StepLabel
      current={0}
      key="2"
      label="Event submission"
      step={2}
    />
    <StepLabel
      current={0}
      key="3"
      label="Done !"
      step={3}
    />
  </div>
  <div
    className="steps-icons"
  >
    <Step
      current={0}
      icon="fa fa-bars"
      step={0}
    />
    <StepSpace
      current={0}
      step={0}
    />
    <Step
      current={0}
      icon="fa fa-microphone"
      step={1}
    />
    <StepSpace
      current={0}
      step={1}
    />
    <Step
      current={0}
      icon="fa fa-calendar-check-o"
      step={2}
    />
    <StepSpace
      current={0}
      step={2}
    />
    <Step
      current={0}
      icon="fa fa-paper-plane"
      step={3}
    />
  </div>
</div>
`;
